garage/misc/wrappers.py

```diff
@@ -0,0 +1,88 @@
+"""Gym env wrappers"""
+import cv2
+import gym
+import numpy as np
+
+from collections import deque
+from gym.spaces import Box
+
+
+class RepeatActionWrapper(gym.core.Wrapper):
```

Comment:
I think it would be best if each of these classes were in its own file.

Generally you should only put more than one class into a file if the classes are all part of the same API, or they are very obviously related.